cloudformation/cfn.yaml

```diff
@@ -0,0 +1,116 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Lambda function to copy team github keys to S3
+Parameters:
+  FunctionName:
+    Type: String
+  TargetBucket:
+    Description: S3 bucket to store the keys in
+    Type: String
+    Default: github-team-keys
+  LambdaSourceBucket:
+    Description: S3 bucket to download the lambda code from
+    Type: String
+    Default: lambda-dist-cross-stream
+  Stack:
+    Type: String
+  AlarmSNS:
+    Description: ARN for an SNS topic, used to send Cloudwatch alerts for things which
+      require immediate action
+    Type: String
+    Default: arn:aws:sns:eu-west-1:743583969668:cloudwatch-alerts
+Resources:
+  KeysToS3LambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service:
+            - lambda.amazonaws.com
+          Action:
+          - sts:AssumeRole
+      Path: "/"
+      Policies:
+      - PolicyName: KeysToS3LambdaPolicy
+        PolicyDocument:
+          Statement:
+          - Effect: Allow
+            Action:
+            - s3:PutObject
+            Resource: !Sub arn:aws:s3:::${TargetBucket}/*
+          - Effect: Allow
+            Action:
+            - dynamodb:Scan
+            Resource: !Sub arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${ConfigTable}
```

Comment:
I still don't understand why you can't get this in a sane way, but `Sub` certainly makes it less horrid.